packages/core/src/browser/tooltip-renderer.tsx

```diff
@@ -0,0 +1,41 @@
+/********************************************************************************
+ * Copyright (C) 2021 Arm and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import * as React from 'react';
+import ReactTooltip from 'react-tooltip';
+import { ReactRenderer } from './widgets/react-renderer';
+import { v4 } from 'uuid';
+
+export const TOOLTIP_ID = v4();
+
+class TooltipRenderer extends ReactRenderer {
```

Comment:
Should the renderer be `injectable` so it can more easily be extended?

Reply:
sure

Reply:
done